src/SDK/Language/Web.php

```diff
@@ -185,6 +185,9 @@ public function getTypeName(array $parameter, array $method = []): string
         if (!empty($parameter['enumValues'])) {
             return \ucfirst($parameter['name']);
         }
+        if (!empty($parameter['name']) && $parameter['name'] === 'queries') {
+            return 'Query[]';
+        }
```

Comment:
I think we might need something like this for Android and Flutter too @lohanidamodar, is there a way to do it without coupling to the parameter name?

Reply:
We need to put something else in the specs. This was my workaround for now